src/build-volume.ts

```diff
@@ -3,20 +3,39 @@ import { AxesHelper, Group, Vector3 } from 'three';
 import { LineBox } from './helpers/line-box';
 import { type Disposable } from './helpers/three-utils';
 
+/**
+ * Represents the build volume of a 3D printer.
+ */
 export class BuildVolume {
+  /** Width of the build volume in mm */
   x: number;
+  /** Depth of the build volume in mm */
   y: number;
+  /** Height of the build volume in mm */
   z: number;
+  /** Color used for the grid */
   color: number;
+  /** List of disposable objects that need cleanup */
   private disposables: Disposable[] = [];
 
+  /**
+   * Creates a new BuildVolume instance
+   * @param x - Width in mm
+   * @param y - Depth in mm
+   * @param z - Height in mm
+   * @param color - Color for visualization (default: 0x888888)
+   */
```

Comment:
There is also an inch mode (that does not impact anything really, except some arc support calculations). It is true that it's `mm` by default, but not necessarily.